verifier/src/test/kotlin/itmo/verifier/KripkeBuildTest.kt

```diff
@@ -0,0 +1,255 @@
+package itmo.verifier
+
+import itmo.verifier.model.Model
+import kotlinx.serialization.modules.SerializersModule
+import kotlinx.serialization.serializer
+import nl.adaptivity.xmlutil.XmlDeclMode
+import nl.adaptivity.xmlutil.serialization.XML
+import org.junit.jupiter.api.Test
+
+
+class KripkeBuildTest {
```

Comment:
Isn't it the same as `SerializationTest.kt`?

Reply:
Not really
It checks that building a model will not cause problems

Reply:
But I don't see diff at all
<img width="400" alt="image" src="https://user-images.githubusercontent.com/31941254/213932366-c36806b5-eb59-436f-a503-6011c294d791.png">

Reply:
Both tests do exactly the same

Reply:
Deleted last 2 lines of serializer test